File: Busqueda_renglones-columnas_03-09-25/matriz.py
matriz = [
    [1,2,3],
    [4,5,6],
    [7,8,9]]

print("Matriz original:")
for i in range(3):
    for j in range(3):
        print(matriz[i][j], end=" ")
    print()

#en filas
for i in range(3):
    for j in range(3):
        print(matriz[j][i], end=" ")

#en columnas
print()
for i in range(3):
    for j in range(3):
        print(matriz[i][j], end=" ")

print()
print("fin del programa")

File: Busqueda_renglones-columnas_03-09-25/matriz.cpp
#include <stdio.h>

int main() {
    int matriz[3][3] = {
        {1, 2, 3},
        {4, 5, 6},
        {7, 8, 9}
    };

    printf("Matriz original:\n");
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < 3; j++) {
            printf("%d ", matriz[i][j]);
        }
        printf("\n");
    }

    // en filas
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < 3; j++) {
            printf("%d ", matriz[j][i]);
        }
       
    }

printf("\n");
    // en columnas
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < 3; j++) {
            printf("%d ", matriz[i][j]);
        }

    }
printf("\n");
    printf("fin del programa\n");
    return 0;
}